images/runtime/Dockerfile

```diff
@@ -3,6 +3,8 @@
 # Copyright 2020-2021 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
+ARG CILIUM_RUNTIME_TAG=2021-08-30
```

Comment:
Note, this provides a default line we can change to rebuild the runtime image when we don't need to make any changes to the build process. This way we can add a "step 0" to the [cilium-runtime image bump steps](https://docs.cilium.io/en/latest/contributing/development/images/#update-cilium-builder-and-cilium-runtime-images) which is just to change this date. This way, when we submit the PR in the subsequent steps, they will actually do something (recalculate git tree shas) rather than doing nothing.

Reply:
For now I didn't document the steps since this is just equivalent to the [v1.10 version](https://github.com/cilium/cilium/pull/17266) of this PR. I think I should follow up with another PR to update the docs so we can backport that correctly to v1.10.

Reply:
^^ I've since found out that the "FORCE_BUILD" env lower down is intended for the same purpose, so this particular change should not be necessary.